src/games/strategy/triplea/ui/BasicGameMenuBar.java

```diff
@@ -1,756 +0,0 @@
-package games.strategy.triplea.ui;
```

Comment:
This class was chopped up. Look for the "extract" commits to see stuff being moved out.